client/src/App.tsx

```diff
@@ -1,10 +1,78 @@
-import Sidebar from './components/Layout/Sidebar';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+
+import { AddUserPage } from './pages/admin/AddUserPage/AddUserPage';
```

Comment:
Would be a good idea to put all exports from `/pages` into `index.ts`, so we don't have twenty import statements

Reply:
Well I usually don't really care about imports so much, especially since editor handles that for you quite well, but sure, if we want to do this way than I will change that

Reply:
Well I usually don't really care about imports so much, especially since editor handles that for you quite well, but sure, if we want to do this way than I will change that

Reply:
Not really necessary, but a nice things to have - especially when IDE messes up with that or for for whatever case you have to change it manually. If you don't have time, just let it be the way it is